_posts/2022-07-25-changelog-175.markdown

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #175"
+date: 2022-07-25 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8767] You can now paste JSON objects into Rust files and convert them to a set of Rust structs (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-175/json" w="700px" %}
+
+* [#9012] Introduce new approach of project model reloading and detection changes in configuration files
+  starting with 2022.2 major IDE release. Now the plugin detects changes in configuration files like
+  `Cargo.toml` even if they are not saved into disk yet, takes into account
+  [Cargo config](https://doc.rust-lang.org/cargo/reference/config.html),
+  [toolchain](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file) and
+  [build script](https://doc.rust-lang.org/cargo/reference/build-scripts.html) files and notifies you if
+  the project model is not up-to-date via floating button in the top-right corner of the editor.
+  You can select preferable way when the project model should be reloaded automatically in
+  `Preferences | Build, Execution, Deployment | Build Tools`. At the same time, Cargo settings were
+  moved to `Preferences | Build, Execution, Deployment | Build Tools | Cargo`
+
+  {% include gif-img.html path="/assets/posts/changelog-175/new-cargo-model-auto-import" w="700px" %}
+
+* [#8868] Highlight macro-machinery-specific syntax in `macro_rules` definitions.
```

Comment:
```suggestion
* [#8868] Highlight macro-machinery-specific syntax in [delcrative macro](https://doc.rust-lang.org/book/ch19-06-macros.html#declarative-macros-with-macro_rules-for-general-metaprogramming) definitions.
```

Reply:
nit: don't like `macro-machinery-specific syntax` collocation. Maybe rephrase it somehow?

Reply:
Can't think of anything better `macro-machinery-specific` right now :(